src/features/completionItemProvider.ts

```diff
@@ -14,6 +14,12 @@ import {CompletionItemProvider, CompletionItem, CompletionItemKind, Cancellation
 
 export default class OmniSharpCompletionItemProvider extends AbstractSupport implements CompletionItemProvider {
 
+    // copied from Roslyn here: https://github.com/dotnet/roslyn/blob/6e8f6d600b6c4bc0b92bc3d782a9e0b07e1c9f8e/src/Features/Core/Portable/Completion/CompletionRules.cs#L166-L169
+    private static DefaultCommitCharacters = [
+        ' ', '{', '}', '[', ']', '(', ')', '.', ',', ':',
+        ';', '+', '-', '*', '/', '%', '&', '|', '^', '!',
+        '~', '=', '<', '>', '?', '@', '#', '\'', '\"', '\\'];
+
```

Comment:
That is one long list 😄 Unsure if all commit characters make sense on all completion item kinds but the API is used as indented.